Source/Scene/ArcGisMapServerImageryProvider.js

```diff
@@ -106,6 +106,24 @@ function ArcGisMapServerImageryProvider(options) {
    */
   this.defaultAlpha = undefined;
 
+  /**
+   * The default alpha blending value on the night side of the globe of this provider, with 0.0 representing fully transparent and
+   * 1.0 representing fully opaque.
+   *
+   * @type {Number}
```

Comment:
```suggestion
   * @type {Number|undefined}
```

Here and below and in every imagery provider. Sorry.

Reply:
I actually thought I did this already, but that was just for `@type {Property}` easy enough.